api/CredentialsContainer.json

```diff
@@ -476,6 +476,74 @@
               "standard_track": true,
               "deprecated": false
             }
+          },
+          "context_option": {
+            "__compat": {
+              "description": "<code>context</code> option",
+              "spec_url": "https://fedidcg.github.io/FedCM/#dom-identitycredentialrequestoptions-context",
+              "support": {
+                "chrome": {
+                  "version_added": "116"
+                },
+                "chrome_android": "mirror",
+                "edge": "mirror",
+                "firefox": {
+                  "version_added": false
+                },
+                "firefox_android": "mirror",
+                "ie": {
+                  "version_added": false
+                },
+                "oculus": "mirror",
+                "opera": "mirror",
+                "opera_android": "mirror",
+                "safari": {
+                  "version_added": false
+                },
+                "safari_ios": "mirror",
+                "samsunginternet_android": "mirror",
+                "webview_android": "mirror"
+              },
+              "status": {
+                "experimental": true,
+                "standard_track": true,
+                "deprecated": false
+              }
+            }
+          },
+          "loginHint": {
```

Comment:
Isn't `loginHint` one level deeper even? Should it be `providers`?

```webidl
dictionary IdentityCredentialRequestOptions {
  sequence<IdentityProviderConfig> providers;
  IdentityCredentialRequestOptionsContext context = "signin";
};

dictionary IdentityProviderConfig {
  required USVString configURL;
  required USVString clientId;
  USVString nonce;
  DOMString loginHint;
};
```

Reply:
I know what you're getting at, but most of `identity.providers` has been supported for as long as `identity`. 

I could perhaps update `<code>context</code> option` to `<code>identity.context</code>`, and `<code>loginHint</code> option` to `<code>identity.providers.loginHint</code>`, but that might not render very nicely. Thoughts?

Reply:
I think those descriptions would be helpful and I have seen longer ones, so I would go for it!

Reply:
OK, done!